eng/resolveContract.targets

```diff
@@ -58,7 +58,7 @@
     <ApiCompatEnableRuleCannotChangeParameterName>true</ApiCompatEnableRuleCannotChangeParameterName>
 
     <_ApiCompatCaptureGroupPattern>.+%5C$([System.IO.Path]::DirectorySeparatorChar)(.+)%5C$([System.IO.Path]::DirectorySeparatorChar)(.+)</_ApiCompatCaptureGroupPattern>
-    <_ApiCompatRuntimePrefixPattern>(.+)/(net%5Cd.%5Cd)-(.+)/(.+)</_ApiCompatRuntimePrefixPattern>
+    <_ApiCompatRuntimePrefixPattern>(.+)/(net%5Cd+%5C.%5Cd)-(.+)/(.+)</_ApiCompatRuntimePrefixPattern>
```

Comment:
hah, I can't wait to discover all the other fun cases like these with `net10.0` 😄

Reply:
You were spot on, see my next commit :D